cypress/integration/ambianic-tests/settings.spec.js

```diff
@@ -34,7 +34,11 @@ context('Settings', () => {
     it("It display connected edge version after connection", () => {
       cy.get('[data-cy=sendRemotePeerID]').click()
 
-      const versionNumber = require('../../../package.json').version
-      cy.get('[data-cy=title-text]').should('contain.text', versionNumber )
+      cy.window().then(win => {
+        win.__store__.commit('PEER_CONNECTED', null)
+
+        const versionNumber = require('../../../package.json').version
```

Comment:
@vickywane what is the purpose of this test? From what I can tell, it checks that the page shows the PWA version number, not the edge device version.

Reply:
I had a previous thinking that the About and Settings page showed the version of the connected edge device.

Reply:
Yes, I realize the previous implementation was confusing. We will correct that with this PR.

Please update this test to check for edge device version in the settings page.